iteration/iteration_test.go

```diff
@@ -328,7 +322,7 @@ func (s *TestIterationRepository) TestCreateIterationSameNameFailsWithinSpace()
 	i3 := iteration.Iteration{
 		Name:       name,
 		SpaceID:    space2.ID,
-		UserActive: &userActive,
+		UserActive: false,
```

Comment:
Remove this line, since `UserActive` will automatically be `false`.

Reply:
Removed it here b92020c